### DISCUSSION TITLE
Tasks added as part of comments should also be picked up

### DISCUSSION THREAD

**Discussion by vishnu-narayanan** [community/community#4886] [Projects and Issues]
Currently, tasks added under the issue description are only picked up for the `tasks` list.  A maintainer or reviewer adding extra tasks as comments will not count towards `tasks` at the moment.

**Comment by Gotterbild** [comment]
Don't think it is a good idea. The discussion can be quite long, contain images, screenshots, screencasts, etc and it will be hard to find those checkboxes. I think it is a good approach to pick up only tasks in the issue description. It will force everybody to add new tasks to the issue description, which I think is a good practice since it will make life easier for everyone.

**Reply by vishnu-narayanan** [reply]
That is a good point. That problem can be solved by something similar to the bitbucket implementation. Tasks added anywhere as a code review comment or a general one will show up at a central location(right side Task section). The reviewer can quickly glance to see if the tasks are complete before doing another review.

**Reply by archon810** [reply]
That's exactly it. Sometimes tasks need to be added by people other than the ticket's creator, and it'd be much better if they were all entered by bug participants and tracked in a central location.

**Reply by Gotterbild** [reply]
Note there's also a moderation issue. 

If anyone can add tasks, issue can turn into a nightmare in case of public repositories that have hundreds of followers.

In my experience any task requires some form is a discussion of at least a confirmation from someone.

There are ways to manage it, via permissions or dialog for issue owner to add task from a comment.
